pkg/sql/sqlstats/insights/registry.go

```diff
@@ -29,10 +31,9 @@ func (r *lockingRegistry) Clear() {
 	r.statements = make(map[clusterunique.ID]*statementBuf)
 }
 
-func (r *lockingRegistry) ObserveStatement(sessionID clusterunique.ID, statement *Statement) {
-	if !r.enabled() {
```

Comment:
Why don't we need this anymore?

Reply:
Ah I think this was left in from a future commit. Put it back!